src/ota/ota_handle.rs

```diff
@@ -457,13 +457,14 @@ where
 
         info!("Rebooting in 5 seconds");
 
-        tokio::time::sleep(std::time::Duration::from_secs(5)).await;
+        if cfg!(not(test)) {
+            tokio::time::sleep(std::time::Duration::from_secs(5)).await;
```

Comment:
`cfg!`, unlike `#[cfg]`, does not remove any code and only evaluates to true or false. [std::cfg](https://doc.rust-lang.org/std/macro.cfg.html)
Keep the version with `#[cfg]`
``` Rust
#[cfg(not(test))]{
   tokio::time::sleep(std::time::Duration::from_secs(5)).await;
   if let Err(error) = crate::power_management::reboot() {
...
}